src/db/Client.ts

```diff
@@ -0,0 +1,21 @@
+import { ConnectionModelType } from './Models';
+
+const { Client } = require('pg');
+
+export default class PgClient {
+  static model: ConnectionModelType;
+
+  public static async newClient(): Promise<any> {
+    return new Client({
+      host: PgClient.model.address,
+      port: PgClient.model.port,
+      password: PgClient.model.password,
+      user: PgClient.model.username,
+      database: 'React',
```

Comment:
Shouldn't be hardcoded. We can use a separate ReactContext for `ActiveDB` or something like that to populate this field

Reply:
Added the ReactContext in a future pull request so this is good for now